domains/ThePlug09 .json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Please rename this file to all lowercase and remove the spaces in the file name.